commit 14d21ac0330de2baeda12b4f3cf68f5a50880041 (HEAD -> master)
Author: Brian Bennett <brian.bennett@joyent.com>
Date:   2019-09-04T10:40:01-07:00 (5 weeks ago)
    
    joyent/manta-thoth#177 Installer should be more clear about which $SDC_ACCOUNT and $SDC_KEY_ID to use
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/bin/sdc-thoth-install b/bin/sdc-thoth-install
index 6d5e7de..bdb0e81 100755
--- a/bin/sdc-thoth-install
+++ b/bin/sdc-thoth-install
@@ -92,8 +92,8 @@ if [[ -z $DOWNLOAD_MANTA ]]; then
 
 fi
 
-[[ -n $SDC_ACCOUNT ]] || fatal "must set \$SDC_ACCOUNT"
-[[ -n $SDC_KEY_ID ]] || fatal "must set \$SDC_KEY_ID"
+[[ -n $SDC_ACCOUNT ]] || fatal "must set \$SDC_ACCOUNT. This is the account in manta that stores your thoth data."
+[[ -n $SDC_KEY_ID ]] || fatal "must set \$SDC_KEY_ID. This must already exist in the $SDC_ACCOUNT account, and needs to be available on this node (ssh-agent is recommended)."
 
 if [[ -z $SDC_URL ]]; then
 	server=us-west-1.api.joyentcloud.com
